src/support/components/FreeAccountSupportOverlay.tsx

```diff
@@ -0,0 +1,35 @@
+import React, {FC, useContext} from 'react'
```

Comment:
I'm not too sure about the file structure here. Right now, I have both Overlays saved in their own folder; `src/support/components`...